sources/dicom/large_image_source_dicom/assetstore/rest.py

```diff
@@ -32,24 +34,38 @@ def _importData(self, assetstore, params):
         parent = ModelImporter.model(parentType).load(params['parentId'], force=True,
                                                       exc=True)
 
+        limit = params.get('limit')
```

Comment:
We want to allow the user to leave this blank, in which case `limit` will probably be an empty string.  The easy way to handle that might be to change this line to `limit = params.get('limit') or None`.

Reply:
Sounds great - thanks David! Do you think our default should also be an empty string? Or is the current default of `100` fine?

Reply:
I think it should be the empty string.